pages/docsprint/onboarding.md

```diff
@@ -0,0 +1,65 @@
+---
+permalink: /docsprint/onboarding/
+---
+# Onboarding Doc Sprint
+
+The theme for 18F's first official [doc sprint](/docsprint) is onboarding. The **Onboarding+Doc Sprint** will culminate on April 17th, 2015.
+
+The ops and onboarding teams are doing tremendous work to make sure new hires have the best possible transition to life at 18F.
+
+There's additional work to do, however, and a key component involves making sure our individual teams and working groups are ready to absorb the upcoming influx of new hires. Also, as the Hub becomes integral to onboarding, we need to make sure the information there is correct and useful.
+
+The onboarding process is more than documentation, but documentation is critical to making sure onboarding can scale.
+
+## The Plan
+
+A large cohort is starting at 18F on April 6th, and these new teammates are the incentive for the onboarding doc sprint. Here's an outline of our thinking.
+
+### Before April 6th
+
+Project teams and working groups get their onboarding houses in order. At a minimum, the documentation for each project should contain the following information:
+
+#### In Project Repository
+
+Make sure README.md has:
+
+* A link to an existing public instance of the project. Links to internal/staging versions can be listed on the Hub (see below) if you don't want them publicly visible.
+* Description of the workflow for submitting project code changes (_e.g._ do you fork or branch?)
+* How to stand up a development version of the project. Installs, scripts, a Vagrantfile for forced fresh install, etc.
+* Project management-related information:
+
+    * links to sprint boards (some are in GitHub, some in Trello, etc.)
+    * links to task management tools (GitHub, Trello, Pivotal)
+    * Quick write-up of where things are: GitHub issues versus sprint plans
+
+Update the CONTRIBUTING file to reflect the workflow as written in README.md.
+
+Tag issues as new-to-project.
+
+#### On the Hub
+
+Make sure your project has a Hub page and that it contains the following information:
```

Comment:
I'm thinking that for every project in `_data/private/projects`, I may auto-create a page in `pages/private` a la https://hub.18f.us/private/18f-site/. Then we can mention that project folks should be able to use that area to add/maintain some of this information.